### PR TITLE
docs: add telegram integration docs [sc-11139]

### PR DESCRIPTION
**Pull request by nahuelon**
## Affected Components
* [ ] Content & Marketing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer
- Add Telegram integration documentation.

> Resolves #[issue-number]

### CODE REVIEW

**Review by pilimartinez — APPROVED** [review]
LGTM, and congrats for this amazing idea 🥳

File: site/content/docs/integrations/telegram.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+menu:
+  docs:
+    parent: "Integrations"
+---
+
+Checkly integrates with [Telegram](https://telegram.org/) and can 
+deliver failure, degradation, and recovery messages to any chat (direct or groups). You can add as many Telegram chats as you wish.
+To enable the Telegram alert channel, take the following steps:
+
+1. First, create an **Telegram Bot** using the [@BotFather](https://t.me/botfather) (detailed instruction [here](https://core.telegram.org/bots)).
```
Comment (stefanjudis) [inline site/content/docs/integrations/telegram.md:13]:
```suggestion
1. Create a **Telegram Bot** using the [@BotFather](https://t.me/botfather) (detailed instruction [here](https://core.telegram.org/bots)).
```

**Review by stefanjudis — APPROVED** [review]
🚢

**Review by tnolet — APPROVED** [review]
LGTM

**Pull request merged**